### PR TITLE
Meetbot_stats

### PR DESCRIPTION
**Pull request by bee2502**
A few lines in script have been tweaked for file writing. Sample Results files(monthwise.txt , weekwise.txt , daywise.txt ) are also updated in the /scripts

### CODE REVIEW

**Comment by decause** [comment]
If you're going to list the dates going back in time, it may be a good idea to output and add the datetime that the script was run at, so that the data has some context?

@ralphbean since you've written the orig, how about a quick peek?

**Comment by ralphbean** [comment]
> it may be a good idea to output and add the datetime that the script was run at, so that the data has some context?

Sure.  That would make sense.

**Comment by pypingou** [comment]
hm, are we mixing tabs and spaces in this file here?

**Comment by ralphbean** [comment]
Friendly tip, @bee2502:  use more specific messages and what you changed in your commit messages.  Otherwise, it is hard to look at the history of commits here (and elsewhere) and quickly know what changed, and why.

File: scripts/meetbot_stats.py [hunk 1]
```diff
@@ -15,12 +15,13 @@ def get_count(start, finish, query):
     query['start'] = start
     query['end'] = end
     response = requests.get(
-        'https://apps.fedoraproject.org/datagrepper/raw',
-        params=query,
+    'https://apps.fedoraproject.org/datagrepper/raw',
+    params=query,
```
Comment (pypingou) [inline scripts/meetbot_stats.py]: Missed one here :)

**Comment by bee2502** [comment]
@ralphbean , @decause : I have updated the indents and edited the code to provide timestamped values. I have also added a sample file for monthwise results "TimeStampedMonthwise.txt". Please look into it.

Other than that - @ralphbean 's earlier code as well as the current script take the current time as input and find daywise/weekwise/monthwise results. Since, the script runs serially, the initial starting time for queries for all topics of a particular type(say monthwise) is not same. (Please check out the last part of 'TimeStampedMonthwise.txt' ) and also some time period is omitted. Do you think we should fix this issue by defining a universal start time for all topics/types and updating the time period accordingly or is this issue negligible ?

@pypingou : Updated the code . Thanks :)

**Comment by ralphbean** [comment]
@decause any thoughts on this?  You get the final say in a merge, I think.

**Comment by decause** [comment]
If you're +1, I'm +1.

:shipit:

**Comment by bee2502** [comment]
@decause , @ralphbean  : If possible, can we wait to merge? I want to add some meetbot topics not mentioned in the docs : https://github.com/fedora-infra/supybot-fedmsg/commit/47339f5741192cb0a2f5b65277cbce0d7c529bed
Documentation : https://fedora-fedmsg.readthedocs.org/en/latest/topics.html#meetbot

**Comment by bee2502** [comment]
Updated the code and results with new topics not available in documentation :
https://github.com/fedora-infra/supybot-fedmsg/commit/47339f5741192cb0a2f5b65277cbce0d7c529bed

**Comment by ralphbean** [comment]
LGTM.  :+1:

**Pull request merged**